identity/src/test/java/com/android/identity/mdoc/util/MdocUtilTest.java

```diff
@@ -0,0 +1,219 @@
+package com.android.identity.mdoc.util;
+
+import com.android.identity.TestVectors;
+import com.android.identity.internal.Util;
+import com.android.identity.mdoc.mso.MobileSecurityObjectParser;
+import com.android.identity.credential.NameSpacedData;
+import com.android.identity.util.CborUtil;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+import co.nstant.in.cbor.model.DataItem;
+
+public class MdocUtilTest {
```

Comment:
I know `MdocUtil.mergeIssuerNamesSpaces` is used in the `DeviceResponseGeneratorTest`, but maybe it would be good to explicitly add a test for it in this tester too? 

Also, I don't see tests for `MdocUtil.generateCredentialRequest` anywhere

Reply:
Yeah, I thought about it but decided against it...  it's not ideal but I think that's an OK answer for now.

Reply:
Thanks for the review, let me push a new commit with the requested changes!